.github/workflows/deploy-docs.yml

```diff
@@ -8,26 +8,95 @@ on:
     paths:
       - '.github/workflows/deploy-docs.yml'
       - 'docs/**'
+    # NOTE: The tag push event is not constrained by branch and path `on` conditions.
+    # For taking a snapshot of the docs state from the tagged commit,
+    # to a versioned subdirectory with dir name matching the git tag substring `v<MAJOR>.<MINOR>`:
+    tags:
+      - 'v[0-9]+.[0-9]+*'
```

Comment:
We only build images and push to dockerhub when the tag is like : `v*.*.*` so we should add the patch level here too?